src/platform/qt/SettingsView.ui

```diff
@@ -97,7 +97,7 @@
    <item row="1" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
-      <number>0</number>
+      <number>1</number>
```

Comment:
Please revert this line. It's a common mistake and I do it myself somewhat frequently.